packages/wallet/src/types.ts

```diff
@@ -1,41 +1,63 @@
 import { EncodeObject } from "@cosmjs/proto-signing";
-import { DeliverTxResponse, SignerData, StdFee } from "@cosmjs/stargate";
+import { Account, DeliverTxResponse, SignerData, SigningStargateClient, StdFee } from "@cosmjs/stargate";
 import { BroadcastTxAsyncResponse, BroadcastTxSyncResponse, Method } from "@cosmjs/tendermint-rpc";
+import { TxRequest } from "@cosmjs/tendermint-rpc/build/comet38";
 import { Tx } from "@demex-sdk/codecs";
+import { DemexSigner } from "./signer";
 
 export type BroadcastTxMode = Method.BroadcastTxAsync | Method.BroadcastTxSync | Method.BroadcastTxCommit;
 
+export type BroadcastTxResult = DeliverTxResponse | BroadcastTxSyncResponse | BroadcastTxAsyncResponse;
+
 export interface PromiseHandler<T> {
   requestId?: string;
   resolve: (result: T) => void;
   reject: (reason?: any) => void;
 }
+
+export type AccountStates = Record<string, AccountState>
```

Comment:
we probably dont need a type just for wrapping a `Record<>`. 
it would be clearer to declare the variable as `accountStates: Record<string, AccountState>`
as opposed to `accountStates: AccountStates`.